### PR TITLE
adding data-lake-customer-facing-diagnostic-files

### PR DESCRIPTION
**Pull request by sap-daniel**
After having this tutorial reviewed by my colleagues and myself I would like to push to production.

### CODE REVIEW

File: tutorials/data-lake-customer-facing-diagnostic-files/data-lake-customer-facing-diagnostic-files.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+FILES_REST_API = '<File Container REST API>'
+CONTAINER = '<File Container ID>-diag'
+CRT_PATH = '<Path to Client Certificate>'
+KEY_PATH = '<Path to Client Key>'
+```
+
+Now, add the logic for using the `LISTSTATUS_RECURSIVE` endpoint, which will list all the items in your directory. You should also format the response so that it is easier to read.
+
+```python
```
Comment (MichaelCzcz) [inline tutorials/data-lake-customer-facing-diagnostic-files/data-lake-customer-facing-diagnostic-files.md:141]: I think it is spelled Python.

File: tutorials/data-lake-customer-facing-diagnostic-files/data-lake-customer-facing-diagnostic-files.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+formatted_data = json.dumps(data, indent=4)
+
+# Print the response and close the connection
+print(formatted_data)
+response.close()
+```
+
+The output should look something like the following.
+
+![Sample REST API output for LISTSTATUS_RECURSIVE](diag-files-8.png)
```
Comment (MichaelCzcz) [inline tutorials/data-lake-customer-facing-diagnostic-files/data-lake-customer-facing-diagnostic-files.md:171]: Add border to image

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+tags: [ tutorial>beginner, software-product>sap-hana-cloud]
+primary_tag: software-product-function>sap-hana-cloud\,-data-lake
+---
+
+## Prerequisites
+ - A SAP HANA Cloud, HANA database instance
+ - A SAP HANA Cloud, data lake instance
+
+## Details
+### You will learn
```
Comment (MichaelCzcz) [inline tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:15]: remove periods

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+   C_ACCTBAL            decimal(15,2)                  not null,
+   C_MKTSEGMENT         varchar(10)                    not null,
+   C_COMMENT            varchar(117)                   not null,
+   primary key (C_CUSTKEY)
+);
+```
+
+Next, right click the HANA database instance in the left side menu and select "Import Data".
+
+![Showing input data link in database explorer.](schedule-2.png)
```
Comment (MichaelCzcz) [inline tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:46]: add border to image

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+-- to create an `EXTERNLOGIN` the HDLRE user needs the `MANAGE ANY USER` privilege.
+GRANT MANAGE ANY USER TO <HDL User>;
+
+-- to create an `EVENT` the HDLRE user needs the `MANAGE ANY EVENT` privilege.
+GRANT MANAGE ANY EVENT TO <HDL User>;
+```
+
+Once the user is created, create a database connection to HDLRE with that user. Right click the HDLRE database in Database Explorer and click properties.
+
+![Right clicking HDLRE database and selecting properties.](schedule-hdl-properties.png)
```
Comment (MichaelCzcz) [inline tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:82]: add border to image

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+
+Copy the host value in the properties modal.
+
+![Selecting and copying the host value in the properties modal.](schedule-copy-host.png)
```
Comment (MichaelCzcz) [inline tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:86]: add border to image

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+
+Close the modal and click the add connection icon in SAP Database Explorer.
+
+![Clicking the add connection icon in Database Explorer](schedule-add-db.png)
```
Comment (MichaelCzcz) [inline tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:90]: add border to image

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+Click OK and then there should be a new database in the left side menu in the Database Explorer. Right click that instance and open a new SQL console. The rest of the tutorial should be completed from there.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 3: ](Set up a Remote Server from SAP HANA Cloud, data lake to SAP HANA Cloud, HANA database)]
+
+The first task in preparing the HDLRE instance is creating a remote server that connects HDLRE to the HDB instance that contains the data you want to access. First, right click the HANA database in Database Explorer and select properties.
+
+![Showing the properties link on the SAP HANA database.](schedule-6.png)
```
Comment (MichaelCzcz) [inline tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:105]: add border to image

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+
+In the properties modal, copy the "Host" value and use it in the below SQL to create a remote server to the HANA database.
+
+![Showing the properties modal and highlighting the Host value](schedule-7.png)
```
Comment (MichaelCzcz) [inline tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:109]: add border to image

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md [hunk 10]
```diff
@@ -0,0 +1,223 @@
...
+
+Since you've already imported your data into your HANA database, you can proceed to preparing HDLRE for scheduling data movement. Use a SQL console that is connected directly to the HDLRE instance.
+
+![Showing opening a new SQL Console for a data lake instance in database explorer.](schedule-5.png)
```
Comment (MichaelCzcz) [inline tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:113]: add border to image

File: tutorials/data-lake-text-search/data-lake-text-search.md [hunk 11]
```diff
@@ -0,0 +1,307 @@
...
+---
+
+## Prerequisites
+ - A provisioned SAP HANA Cloud, data lake
+ - Access to the SAP HANA Cloud, data lake File Store
+ - Access to the SAP Database Explorer
+ - An installation of the HDLFSCLI
+
+## Details
+### You will learn
```
Comment (MichaelCzcz) [inline tutorials/data-lake-text-search/data-lake-text-search.md:17]: remove periods

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Thank you for creating this pull request. There are some style issues that need to be fixed. I haven't commented all occurrences, so please check all tutorials in this PR and make the necessary changes.
For more information, please check the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9).

File: tutorials/data-lake-customer-facing-diagnostic-files/data-lake-customer-facing-diagnostic-files.md [hunk 12]
```diff
@@ -0,0 +1,248 @@
...
+---
+
+## Prerequisites
+ - Access to the Cloud Foundry space of a SAP HANA Cloud, data lake
+ - Installation of the HANA data lake File Store Command Line Interface (HDLFSCLI)
+ - [Optional] Completion of [Getting Started with Data Lake Files HDLFSCLI tutorial](group.hana-data-lake-containers)
+ - Access to a licensed SAP HANA Cloud, data lake instance
+
+## Details
+### You will learn
```
[the remark this answers is not in the document]
Comment (sap-daniel) [reply tutorials/data-lake-customer-facing-diagnostic-files/data-lake-customer-facing-diagnostic-files.md:17]: Done.

File: tutorials/data-lake-customer-facing-diagnostic-files/data-lake-customer-facing-diagnostic-files.md (hunk 2 above)
Comment (sap-daniel) [reply tutorials/data-lake-customer-facing-diagnostic-files/data-lake-customer-facing-diagnostic-files.md:171]: Done.

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md (hunk 3 above)
Comment (sap-daniel) [reply tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:15]: Done.

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md (hunk 5 above)
Comment (sap-daniel) [reply tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:82]: Done.

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md (hunk 6 above)
Comment (sap-daniel) [reply tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:86]: Done.

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md (hunk 7 above)
Comment (sap-daniel) [reply tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:90]: Done.

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md (hunk 8 above)
Comment (sap-daniel) [reply tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:105]: Done.

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md (hunk 9 above)
Comment (sap-daniel) [reply tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:109]: Done.

File: tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md (hunk 10 above)
Comment (sap-daniel) [reply tutorials/data-lake-schedule-data-movement/data-lake-schedule-data-movement.md:113]: Done.

File: tutorials/data-lake-text-search/data-lake-text-search.md (hunk 11 above)
Comment (sap-daniel) [reply tutorials/data-lake-text-search/data-lake-text-search.md:17]: Done.

**Review by sap-daniel — COMMENTED** [review]
Made all of the requested changes.

**Pull request merged**